src/ext/java/org/rutebanken/tiamat/ext/fintraffic/auth/FintrafficSecurityConfig.java

```diff
@@ -0,0 +1,25 @@
+package org.rutebanken.tiamat.ext.fintraffic.auth;
+
+import org.rutebanken.tiamat.auth.AuthorizationService;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Profile;
+import org.springframework.security.config.annotation.method.configuration.EnableMethodSecurity;
+
+@Configuration
+@Profile("fintraffic")
+@EnableMethodSecurity(prePostEnabled = true)
+public class FintrafficSecurityConfig {
+
+  @Bean
+  AuthorizationService authorizationService(
+    @Value("${tiamat.ext.fintraffic.security.oidc-server-uri}") String oidcServerUri,
+    @Value("${tiamat.ext.fintraffic.security.client-id}") String clientId,
+    @Value("${tiamat.ext.fintraffic.security.client-secret}") String clientSecret,
+    @Value("${tiamat.ext.fintraffic.security.enable-codespace-filtering:false}") boolean enableCodespaceFiltering
```

Comment:
We can't enable this just yet because we don't have an exhaustive set of codespaces for Finnish providers.